tools/ci_build/build.py

```diff
@@ -480,6 +480,8 @@ def setup_tensorrt_vars(args):
                              "tensorrt_home='{}' valid={}."
                              .format(tensorrt_home, tensorrt_home_valid))
 
+        os.environ["TENSORRT_MAX_BATCH_SIZE"] = "13"
```

Comment:
add a comment? 
btw, why 13? is that the maximum batch size that we test in the unit tests?
if someone adds a test which requires larger, let's make it easy for them to figure out why.